litho-core/src/main/java/com/facebook/litho/DebugComponent.java

```diff
@@ -260,6 +260,22 @@ public String getTestKey() {
     return isLayoutNode() ? mNode.getTestKey() : null;
   }
 
+  /**
+   * Returns this component's testKey or null if none is set.
+   *
+   * <p>Unlike {@link #getTestKey()}, this function can return a test key set on any Component,
+   * including container Components which resolve into LayoutNodes.
+   *
+   * <p>Unlike {@link #getTestKey()}, this function can also return test keys set on individual
+   * Components even when they are all resolved into a single InternalNode.
+   */
```

Comment:
Can you please add a similar comment to `getTestKey()` method, so that users of it would know the quirks too?

Reply:
Sure, sounds good. Done

Reply:
Thank you!